_artifacts/document-attributes-global.adoc

```diff
@@ -69,3 +69,5 @@ endif::[]
 :osd-latest-version: 4
 :rosa-latest-version: 4
 :os-latest-version: 1.32
+:rhelai-productname-long: Red{nbsp}Hat Enterprise Linux AI
+:rhelai-productname-short: RHELAI
```

Comment:
I think this should be:

:rhelai-productname-short: RHEL AI

Reply:
```suggestion
:rhelai-productname-short: RHEL AI
```
You're right, thank you!